### DISCUSSION TITLE
Unable to acces Influxdb CLI in hassio-addon

### DISCUSSION THREAD

**Discussion by geelenbert** [hassio-addons/addon-influxdb#131] [Q&A]
I'm unable to get into the influxdb CLI of the addon.
I want to execute some database commands for clearing old data and copying old data to new entity_id's. 

There was some sort of old work around to do this using the docker command and the SSH plugin.
https://community.home-assistant.io/t/how-to-access-influxdb-cli-form-hassio/78915

But at this point, due to the restrictions on the SSH addon, you can't get acces to the root system trough ssh.

How should it be possible to execute CLI commands on the influx database?

**Comment by frenck** [comment]
The Community SSH & Web Terminal add-on has the influxdb cli available. It can be used to connect to the influxdb add-on.

**Reply by Gilles-Clemenceau** [reply]
How can I do that ? I'm looking everywhere but I can't fin a way to connect to influxdb via cli.
